### PR TITLE
0.6.2 release prep

### PR DESCRIPTION
**Pull request by Frigyes06**

### CODE REVIEW

**Comment by Frigyes06** [comment]
> Do we wait for the pending PRs to be merged or not?

Package changes should target main. I say we release now. Don't want toess with release notes

File: Cargo.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [package]
 name = "uad_gui"
 description = "A cross-platform GUI debloater for android devices"
-version = "0.6.1"
+version = "0.6.2"
 authors = ["Universal-Debloater-Alliance"]
 license = "GPL-3.0"
 homepage = "https://github.com/Frigyes06/universal-android-debloater"
```
Comment (ProtonOSS) [inline Cargo.toml:7]: Owner has changes should `homepage`, and `repository` be updated to the https://github.com/Universal-Debloater-Alliance/universal-android-debloater right?
```suggestion
homepage = "https://github.com/Universal-Debloater-Alliance/universal-android-debloater"
repository = "https://github.com/Universal-Debloater-Alliance/universal-android-debloater"
```

Comment (AnonymousWP) [reply Cargo.toml]: Well spotted, thanks.

**Comment by Frigyes06** [comment]
These patches need to go out asap, as they contain important fixes.

**Comment by AnonymousWP** [comment]
> These patches need to go out asap, as they contain important fixes.

I am waiting for the people to review my proposals/suggestions.

**Pull request merged**